ci-operator/config/openshift/aws-load-balancer-operator/openshift-aws-load-balancer-operator-main.yaml

```diff
@@ -49,6 +64,31 @@ tests:
     make test
   container:
     from: src
+- as: e2e-aws-operator
+  steps:
+    cluster_profile: aws
+    dependencies:
+      OO_INDEX: ci-index-aws-load-balancer-operator-bundle
+    env:
+      OO_CHANNEL: alpha
+      OO_INSTALL_NAMESPACE: aws-load-balancer-operator
+      OO_PACKAGE: aws-load-balancer-operator
+      OO_TARGET_NAMESPACES: '!install'
+    post:
+    - chain: ipi-aws-post
```

Comment:
These steps are explicitly given, since we are using a modified version of an existing workflow ([optional-operators-ci-aws](https://github.com/openshift/release/blob/master/ci-operator/step-registry/optional-operators/ci/aws/optional-operators-ci-aws-workflow.yaml)).

Reply:
I see, then I agree with [Arjun's point](https://github.com/openshift/release/pull/27205#discussion_r838617772) on the reason of why we need to specify the workflow.